force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1697,6 +1697,14 @@
         <shortDescription>Error for DIs with populated RD fields when RD2 is disabled.</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can process Recurring Donations.</value>
     </labels>
+    <labels>
+        <fullName>RD2_nextDonation</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for next donation mapping.</shortDescription>
+        <value>Paid Installments: {0}/{1},&lt;/br&gt; Next Installment Date: {2}.</value>
```

Comment:
I recommend this as two labels (one of which is probably already the label of a field). I also recommend not using merge fields nor including the `</br>` tag. This can be rendered in the UI as two separate fields, such as:
```
<label.RD2_PaidInstallments>: value
<label.npe03__Recurring_Donation__c.NextDonationDate.label>: value
```
That will greatly help with translations and formatting.

Reply:
I appreciate the suggestion! the br has already been deleted, since for this case there is a logic that allows modifying the label from the admin and that this be reflected directly in the component in a dynamic way taking into account the line breaks, the design requirements and the ellipses of the paragraphs